packages/library-base/src/androidMain/kotlin/io/realm/kotlin/internal/platform/SystemUtilsAndroid.kt

```diff
@@ -36,3 +39,12 @@ public actual fun assetFileAsStream(assetFilename: String): InputStream = try {
 // Returns the default logger for the platform
 public actual fun createDefaultSystemLogger(tag: String, logLevel: LogLevel): RealmLogger =
     LogCatLogger(tag, logLevel)
+
+public actual fun currentTime(): RealmInstant {
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
We should document that there is some precision loss depending on the SDK version used.